packages/builders/src/components/selectMenu/MentionableSelectMenu.ts

```diff
@@ -31,4 +38,55 @@ export class MentionableSelectMenuBuilder extends BaseSelectMenuBuilder<APIMenti
 	public constructor(data?: Partial<APIMentionableSelectComponent>) {
 		super({ ...data, type: ComponentType.MentionableSelect });
 	}
+
+	/**
+	 * Adds default roles to this auto populated select menu.
+	 *
+	 * @param roles - The roles to add
+	 */
+	public addDefaultRoles(...roles: RestOrArray<Snowflake>) {
+		const normalizedValues = normalizeArray(roles).map((id) => ({
+			id,
+			type: SelectMenuDefaultValueType.Role as const,
+		}));
+
+		this.data.default_values ??= [];
+		optionsLengthValidator.parse(this.data.default_values.length + normalizedValues.length);
+		this.data.default_values.push(...normalizedValues);
+		return this;
+	}
+
+	/**
+	 * Adds default users to this auto populated select menu.
+	 *
+	 * @param users - The users to add
+	 */
+	public addDefaultUsers(...users: RestOrArray<Snowflake>) {
+		const normalizedValues = normalizeArray(users).map((id) => ({
+			id,
+			type: SelectMenuDefaultValueType.User as const,
+		}));
+
+		this.data.default_values ??= [];
+		optionsLengthValidator.parse(this.data.default_values.length + normalizedValues.length);
+		this.data.default_values.push(...normalizedValues);
+		return this;
+	}
+
+	/**
+	 * Sets default values to this auto populated select menu.
+	 *
+	 * @param values - The values to set
+	 */
+	public setDefaultValues(
```

Comment:
Why is there no `addDefaultValues`?

Reply:
Resolved with 1126cd91c4fe9d7910a364a593913de7d87b78fe.